src/Chirp.Infrastructure/Models/AuthorFollower.cs

```diff
@@ -8,11 +8,11 @@ public class AuthorFollower
     public required string FollowerId { get; set; }
     
     [Required]
-    public required Author Follower { get; set; }
+    public Author Follower { get; set; }
     
     [Required]
     public required string FollowingId { get; set; }
     
     [Required]
-    public required Author Following { get; set; }
+    public Author Following { get; set; }
```

Comment:
Why is required removed here? Even though we keep the Required data annoint?

Reply:
I see it this way. A Cheep needs an Author, but an Author does not need to be passed in when creating a Cheep. I have done this since we don't need to pass an Author for the navigation property to work. So we can still use cheep.Author even if we only pass in AuthorId and not Author on Cheep creation. There may be times where we want to pass in Author on cheep creation, but we don't have to. That is why I removed required from public required Author Author { get; set; }. So that adding Author on Cheep creation is possible, but not required. However a cheep still needs to have an author no matter what which is why I kept [Required].